pkg/commands/test/main.go

```diff
@@ -0,0 +1,51 @@
+package main
```

Comment:
What is the point of this file?  
Why not put it inside a `someFile_test.go`?

Reply:
whoops that was accidentally committed